File: app/src/main - 副本/java/com/zmcursor/mydaliy/Model/Bean/Latest.java
package com.zmcursor.mydaliy.Model.Bean;

import java.util.List;

/**
 * Created by ZMcursor on 2018/5/30 0030.
 */

public class Latest {
    public String date;
    public List<Story> stories;
    public List<TopStory> top_stories;

    @Override
    public String toString() {
        return "Latest{" +
                "date='" + date + '\'' +
                ", stories=" + stories +
                ", top_stories=" + top_stories +
                '}';
    }
}

File: app/src/main/java/com/zmcursor/mydaliy/Model/Bean/Latest.kt
package com.zmcursor.mydaliy.Model.Bean

/**
 * Created by ZMcursor on 2018/5/30 0030.
 */

class Latest {
    var date: String = ""
    var stories: List<Story>? = null
    var top_stories: List<TopStory>? = null

    override fun toString(): String {
        return "Latest{" +
                "date='" + date + '\'' +
                ", stories=" + stories +
                ", top_stories=" + top_stories +
                '}'
    }
}
